components/layout/LocaleDropdown.js

```diff
@@ -1,8 +1,11 @@
 import { PageContext } from '../../pages/_app';
+import useDotCMSApi from '../../hooks/useDotCMSApi';
+import { getLanguages } from '../../utils/dotcms';
+
 import React from 'react';
 
 const Options = ({ languages }) => {
```

Comment:
not sure if this is the right moment to do this, but we should start adding `PropTypes`
https://reactjs.org/docs/typechecking-with-proptypes.html